deployment-scripts/helm-charts/deepfence-console/templates/database/neo4j/neo4j.yaml

```diff
@@ -63,12 +63,16 @@ spec:
           volumeMounts:
             - name: "neo4j"
               mountPath: /data
+              subPath: "data"
```

Comment:
This is puzzling, I don't fully get why this is needed

Reply:
https://stackoverflow.com/questions/65399714/what-is-the-difference-between-subpath-and-mountpath-in-kubernetes

Reply:
Still don't fully grasp what issues was causing data loss.
The way I understand it is, without subPath, the volume could not be shared across multiple `mountPath` but with subPath it can?

Reply:
After offline discussion, it makes sense.
For record, in docker we used to have 1 volume per mount point but for k8s we have 1 volume shared across all mount points.
Now, without `subPath`, we were attaching the same volume at different hook points, messing up the architecture for each `subPath`